.github/workflows/ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.4"
           - "8.0"
+          - "8.1"
```

Comment:
Can you add the Symfony version to the build matrix? This was also done here: https://github.com/byWulf/apitk-common-bundle/pull/15/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f